roles/container-engine/containerd/tasks/main.yml

```diff
@@ -56,6 +56,14 @@
     - containerd-shim-runc-v2
     - ctr
 
+- name: containerd ｜ Create certs.d directories
+  file:
+    path: "{{ containerd_cfg_dir }}/{{ item }}"
+    state: directory
+    mode: 0755
+  with_items:
+    - certs.d
+
```

Comment:
This task could be unnecessary if specifying `recurse: true` on the task `name: containerd ｜ Create registry directories`.

Reply:
Thanks you suggest, done.